basic/src/test/java/org/jboss/hal/testsuite/test/configuration/logging/LogCategoriesTestCase.java

```diff
@@ -51,7 +51,6 @@ public static void tearDown() throws IOException, TimeoutException, InterruptedE
         operations.removeIfExists(LOGGER_ADDRESS);
         operations.removeIfExists(LOGGER_TO_BE_ADDED_ADDRESS);
         operations.removeIfExists(LOGGER_TO_BE_REMOVED_ADDRESS);
-        administration.reloadIfRequired();
```

Comment:
Reload is unnecessary since it's performed again in parent `AfterClass` method. Order of `AfterClass` invocations is guaranteed by JUnit Javadoc.